packages/react-components/react-toast/stories/Toast/Offset.stories.tsx

```diff
@@ -19,19 +19,56 @@ export const Offset = () => {
 
   return (
     <>
+      <Field label="Horizontal offset">
+        <SpinButton
+          value={horizontal}
+          onChange={(e, data) => {
+            console.log(data.value);
+            if (data.value) {
+              setHorizontal(data.value);
+            } else if (data.displayValue !== undefined) {
+              const newValue = parseFloat(data.displayValue);
+              if (!Number.isNaN(newValue)) {
+                setHorizontal(newValue);
+              }
+            }
+          }}
+        />
+      </Field>
+      <Field label="Vertical offset">
+        <SpinButton
+          value={vertical}
+          onChange={(e, data) => {
+            console.log(data.value);
+            if (data.value) {
+              setVertical(data.value);
+            } else if (data.displayValue !== undefined) {
+              const newValue = parseFloat(data.displayValue);
+              if (!Number.isNaN(newValue)) {
+                setVertical(newValue);
+              }
+            }
+          }}
+        />
+      </Field>
+      <br />
+      <Button onClick={() => notify('bottom-start')}>bottom-start</Button>
+      <Button onClick={() => notify('bottom-end')}>bottom-end</Button>
+      <Button onClick={() => notify('top-start')}>top-start</Button>
+      <Button onClick={() => notify('top-end')}>top-end</Button>
```

Comment:
<img width="539" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/9af668f9-35bf-4ea8-9b5c-03ab9239616e">

Not sure that it's clear what it does, better 🔽

```suggestion
      <Button onClick={() => notify('bottom-start')}>Create at "bottom-start"</Button>
      <Button onClick={() => notify('bottom-end')}>Create at "bottom-end"</Button>
      <Button onClick={() => notify('top-start')}>Create at "top-start"</Button>
      <Button onClick={() => notify('top-end')}>Create at "top-end"</Button>
```

Reply:
Updated to have single button and radio selection for position